### ISSUE TITLE
`History item reference: main` polish

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#238984]
Testing #238850

Could we instead say `Branch: main` here to keep it concise for screen reader users?

**Comment by lszomoru** [comment]
The challenge here is that the accessibility help dialog is rendered in core so we cannot use source control provider specific terminology (ex: branch, commit, etc). "History Item Reference" is the current terminology that we are using but this may change as we finalize the history provider API.

**Comment by meganrogge** [comment]
Makes sense, feel free to close
